### PR TITLE
Deployment for Feeding America

### PR DESCRIPTION
**Pull request by jehangiramjad**

### CODE REVIEW

File: run_test.sh [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 set -e
+export NODE_OPTIONS=--openssl-legacy-provider
```
Comment (shifucun) [inline run_test.sh]: can remove this now

File: server/configmodule.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ class LocalPrivateConfig(PrivateConfig):
     AI_CONFIG_PATH = os.path.abspath(
         os.path.join(os.path.curdir, '..', 'deploy/overlays/local/ai.yaml'))
     LOCAL = True
-    SECRET_PROJECT = 'datcom-website-private'
+    SECRET_PROJECT = 'datcom-feedingamerica'
```
Comment (shifucun) [inline server/configmodule.py:107]: Can you create new classes like LocalIitmConfig? Can check what is changed in this PR https://github.com/datacommonsorg/website/pull/1497/files

File: run_test.sh (hunk 1 above)
Comment (jehangiramjad) [reply run_test.sh]: already done in the latest commit :)

File: server/configmodule.py (hunk 2 above)
Comment (jehangiramjad) [reply server/configmodule.py:107]: It already exists for feeding america. Do they need to be modified?

File: deploy/overlays/feeding-america/kustomization.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Kustomization for private website running on GCP `datcom-website-private` project.
+# Kustomization for the Feeding America deployment.
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```
Comment (shifucun) [inline deploy/overlays/feeding-america/kustomization.yaml:18]: should change line 20 to "-feedingamerica"

File: server/configmodule.py [hunk 4]
```diff
@@ -104,7 +107,7 @@ class LocalPrivateConfig(PrivateConfig):
     AI_CONFIG_PATH = os.path.abspath(
         os.path.join(os.path.curdir, '..', 'deploy/overlays/local/ai.yaml'))
     LOCAL = True
-    SECRET_PROJECT = 'datcom-website-private'
+    SECRET_PROJECT = 'datcom-website-private '
```
Comment (shifucun) [inline server/configmodule.py:110]: revert this change?

File: deploy/overlays/feeding-america/kustomization.yaml [hunk 5]
```diff
@@ -12,12 +12,12 @@
...
 
-nameSuffix: -private
+nameSuffix: feedingamerica
```
Comment (shifucun) [inline deploy/overlays/feeding-america/kustomization.yaml:20]: -feedingamerica, missing "-" here

**Review by beets — COMMENTED** [review]
can we also create a local config to run this deployment? local-feedingamerica? thank you!
